tests/jerry-test-suite/es2015/22/22.02/22.02.01/22.02.01-021.js

```diff
@@ -0,0 +1,22 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var a = new Float32Array([0.1, 0.2, 0.3]);
```

Comment:
Because I thought this case can reflect more than that in the original issue.
Yes, I will add the reported script to the tests/jerry. There is no es2015 subdir in tests/jerry yet, I will find a way to add them.